onnxruntime/core/providers/cpu/tensor/upsample.cc

```diff
@@ -342,7 +342,7 @@ Status Upsample<T>::BaseCompute(OpKernelContext* context, const std::vector<floa
   Tensor* Y = context->Output(0, Y_dims);
 
   if (no_scale) {
-    memcpy(Y->MutableDataRaw(), X->DataRaw(), Y->Size() * sizeof(T));
+    memcpy(Y->MutableDataRaw(), X->DataRaw(), Y->Size());
```

Comment:
Should we rename Tensor::Size to Tensor::SizeInBytes in a separate PR?

Reply:
> Should we rename Tensor::Size to Tensor::SizeInBytes in a separate PR?

It comes with a cost. You know, it's a source code level break change. Many partners are using this class.

Reply:
Thanks for the fixing. Didn't know Size() returns byte size.

Reply:
I'd prefer making the names more explicit at the cost of possibly breaking some ongoing code. Otherwise, someone else would make the same mistake later and it's more expensive to debug and fix then.

Reply:
Agree. Let me make the change in a separate PR.